Toolkit.py

```
@@ -1,321 +1,321 @@
+		if (not os.path.isfile('./musync_data/SongName.json') or (cls.__resourceFileInfo["SongName"]["Version"] > SongName.Version())):
+			info:dict[str,any] = cls.__resourceFileInfo["SongName"];
+			Toolkit.ResourceReleases(info["offset"], info["lenth"], './musync_data/SongName.json');
+		# 检查mscorlib.dll是否存在
+		logger.debug("Check \"mscorlib.dll\" is not exists...");
+		if (not os.path.isfile("./mscorlib.dll") or (Toolkit.GetHash('./mscorlib.dll') != cls.__resourceFileInfo["CoreLib"])):
+			info:dict[str,any] = cls.__resourceFileInfo["CoreLib"];
+			Toolkit.ResourceReleases(info["offset"], info["lenth"], "./mscorlib.dll");
+		# 检查字体文件是否存在
+		logger.debug("Check \"musync_data\\LXGW.ttf\" is not exists...");
+		if not os.path.isfile('./musync_data/LXGW.ttf' or (Toolkit.GetHash('./musync_data/LXGW.ttf') != cls.__resourceFileInfo["Font"])):
+			info:dict[str,any] = cls.__resourceFileInfo["Font"];
```

Ruff (0.14.8)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)